enhancements/baremetal/add-bare-metal-host-to-vsphere.md

```diff
@@ -0,0 +1,386 @@
+---
+title: add-bare-metal-host-to-vsphere
+authors:
+  - "rvanderp3"
+reviewers: 
+  - "gnufied"
+  - "JoelSpeed"
+approvers: 
+  - "JoelSpeed"
+api-approvers: 
+  - None
+creation-date: 2025-02-10
+last-updated: 2025-02-10
+tracking-link: 
+  - https://issues.redhat.com/browse/OCPSTRAT-1917
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+# Bare Metal Hosts on vSphere Integrated Cluster
+
+## Summary
+
+As customers begin to use alternate virtualization or deployment patterns in their cluster they may need the ability to provision bare metal hosts along side their cloud integrated nodes. This is needed if a customer is unable to readily migrate their workloads to another OpenShift cluster or if they have a specific need to schedule workloads to both integrated and non-integrated nodes.
+
+## Motivation
+
+### User Stories
+
+As a maintainer of an OpenShift cluster I need the ability to add bare metal hosts to my integrated OpenShift installation so I can migrate workloads to bare metal hosts without having to reinstall the cluster.
+
```

Comment:
As discussed, unless otherwise workloads are modified or BM nodes have special taints, nothing will prevent scheduling of pods that require storage to BM nodes even though they have no CSI driver installed. And that will cause those workloads to get stuck. We need to keep that in mind.